frontend/test/__support__/e2e/helpers/e2e-embedding-helpers.js

```diff
@@ -3,12 +3,78 @@ import { METABASE_SECRET_KEY } from "__support__/e2e/cypress_data";
 const jwtSignLocation =
   "frontend/test/__support__/e2e/external/e2e-jwt-sign.js";
 
-export function visitEmbeddedPage(payload) {
-  const stringifiedPayload = JSON.stringify(payload);
+/**
+ * Programatically generate token and visit the embedded page for question or dashboard
+ *
+ * @param {object} payload
+ * @param {{setFilters: string, hideFilters:string}}
+ *
+ * @example
+ * visitEmbeddedPage(payload, {
+ *   // We divide filter values with an ampersand
+ *   setFilters: "id=92&source=Organic",
+ *   // We divide multiple hidden filters with coma.
```

Comment:
Just noticed this horrendous typo!
Let's hope nobody will end up in coma :D

I'll fix it in one of the next PRs in order to avoid waiting for the whole CI to run again.